record-two/record.controller.js

```diff
@@ -3,8 +3,19 @@
 
     angular.module('chaise.record')
 
-    .controller('RecordController', ['$rootScope', function RecordController($rootScope) {
-        // made this for manipulating code in html for record app
+    .controller('RecordController', ['$window', function RecordController($window) {
+        var vm = this;
 
+        vm.editRecord = chaiseConfig.editRecord;
+
+        vm.createRecord = function() {
+            var hash = $window.location.hash;
+            // Should I substring based on the position of id or should I split on '/' and piece back together parts 0,1?
+            $window.location.href = "../recordedit/" + hash.substring(0, hash.indexOf('id')-1);
```

Comment:
I don't think we can assume `"id"` will always be available, splitting on '/' is better. The column name could be "ID" or "identifier" or "some_other-name", no?